src/comps/management/comps/productsMng/comps/addProductsCeo/comps/AddArealProduct.jsx

```diff
@@ -2,6 +2,8 @@ import React, { useState } from "react";
 import useAddProductToDB from "../../../../../../../outSideFunction/functionMng/addProduct";
 import useCheckNameExixt from "../../../../../../../outSideFunction/functionMng/checkIfNameExixtProduct";
 import productMngCss from "../../../../../../../styles/productsMng.module.css";
+import { PRODUCTS } from "../../../../../../../constans/hardCoded/mangement/productsMng/products";
+import { validateImageUrl } from "../../../../../../../outSideFunction/checkImage/ImageValidation";
 export default function AddArealProduct({ setShowAddProduct }) {
```

Comment:
You are importing this from some path containing 'outSideFunctin' what is this outside function?